src/js/helpers/cleanData.js

```diff
@@ -507,73 +470,6 @@ export const cleanNavigation = (navigation, lang) => {
   return cleanedNavigation;
 };
 
-const checkUrl = async url => {
-  return await axios({
-    method: 'HEAD',
-    url: url,
-  })
-    .then(res => url)
-    .catch(error => null);
-};
-
-const getWorkingDocumentLink = async filename => {
-  // Single source mode, example use case:
-  // If we're on PROD, we should only get prod documents
-  if (process.env.CMS_DOCS !== 'multiple') {
-    return `${process.env.CMS_DOCS}/${filename}`;
-  }
-
-  // Multi source mode, let's do some url checking and get something
-  // that works. example use case:
-  // If we're on STAGING, we want docs imported from PROD to work,
-  // as well as any new docs we added when testing on staging
-  if (process.env.CMS_DOCS === 'multiple') {
-    const docUrls = [
-      'https://joplin-austin-gov-static.s3.amazonaws.com/production/media/documents',
-      'https://joplin-austin-gov-static.s3.amazonaws.com/staging/media/documents',
-    ];
```

Comment:
Oh wow this is amazing. Can we finally get rid of the weird CMS_DOCS logic stuff?